vcd/metadata.go

```diff
@@ -369,6 +384,32 @@ func updateMetadataInState(d *schema.ResourceData, vcdClient *VCDClient, resourc
 	return nil
 }
 
+// filterAndSetVcdMetadataEntryToVm filters the metadata created by VCD from the input metadata, that gets updated without these.
```

Comment:
> , that gets updated without these.

Please rephrase this comment. It's hard to grasp what exactly it means :)

Also, would be good to emphasize that the inherited metadata entries are actually "removed" from the metadata.

Reply:
Yep, now that I re-read it after a few days I can say it was a bit odd. I had a few iterations of rephrasing, I think it's better now.

Reply:
Thanks! I noticed a typo, but I would even simplify it further:

> The filtered out entries **will not be longer present** in the original metadata input parameter.

->

> Note: the filtered out entries **are removed from** the original metadata input parameter.

Reply:
Changed!